src/translations/en.json

```
@@ -248,7 +248,8 @@
         "cardFrozen": "That card is frozen! You must wait a turn to play it.",
         "playFromHand": "You can only play a card that is in your hand",
         "blockedByQueen": "Your opponent's queen prevents you from targeting their other cards",
-        "blockedByMultipleQueens": "You cannot play a Targeted One-Off (Two, Nine) when your opponent has more than one Queen"
+        "blockedByMultipleQueens": "You cannot play a Targeted One-Off (Two, Nine) when your opponent has more than one Queen",
+        "notInMainPhase" : "You can only play that card in a main phase"
```

Great! Can you please update the existing `validate` functions to throw this same error?

Let's also reword it to not mention playing a card, because this also applies to drawing and passing
```suggestion
        "notInMainPhase" : "You can only do that in the main phase"
```

Please make the same revision in the other language files as well. This will also require updating 
```
it('Displays "Opponent May Counter" when player had neither glasses nor a queen')
```
in `0_countering.spec.js` because it makes an assertion about the error message that comes back, so we'll need to write this message instead. in the assertion of the error